content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/mtls_test.sh

```diff
@@ -75,6 +75,7 @@ _verify_contains snip_configure_mutual_tls_origination_for_egress_traffic_5 "GET
 
 # @cleanup
 set +e # ignore cleanup errors
+kubectl label namespace default istio-injection- || true
```

Comment:
Is `|| true` needed even though it's running after `set +e`?

Actually, I wonder if the `set +e` should be automatically added by the frameworks, rather than first line of every test? WDYT?

Reply:
> Is || true needed even though it's running after set +e?

It's not, but it doesn't hurt

> Actually, I wonder if the set +e should be automatically added by the frameworks, rather than first line of every test? WDYT?

I'm probably fine with that, but would rather not change it in this PR.

Reply:
OK, I'll do that in a followup PR.